### PR TITLE
Chores and fixes

### PR DESCRIPTION
**Pull request by flmel**
- Adds the missing NEPs and their credits to Readme.md
- Changes the Statuses of the approved and merged NEPs to `Final`
- Ensures number/naming consistency across files and NEP headers
- Updates NEP headers to match the preset on [NEP-0001](./neps/nep-0001.md)

### CODE REVIEW

**Review by frol — APPROVED** [review]
@flmel Thank you for taking care of these!

**Comment by frol** [comment]
@race-of-sloths score 5

**Comment by race-of-sloths** [comment]
@flmel Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
Are you going to win race-of-sloths this month? If so, you should speed up! Run, Sloth, run!

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://race-of-sloths.com/images/bot-banner-dark.svg">
    <source media="(prefers-color-scheme: light)" srcset="https://race-of-sloths.com/images/bot-banner-light.svg">
    Shows inviting banner with latest news.
</picture>](https://race-of-sloths.com/latest-news)

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/flmel?type=bot&pr=near/NEPs/566&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/flmel?type=bot&pr=near/NEPs/566&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/flmel)


<details><summary>Current status: <i>executed</i></summary>

| Reviewer | Score |
|--------|--------|
| @frol  | 5 |


Your contribution is much appreciated with a final score of 5!
You have received 65 (50 base + 15 weekly bonus) Sloth points for this contribution


Another weekly streak completed, well done @flmel! To keep your weekly streak and get another bonus make pull request next week! Looking forward to see you in race-of-sloths
</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/flmel)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Invite contributor** `@race-of-sloths invite` to invite the contributor to participate in a race or include it, if it's already a runner.
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

**Pull request merged**
